control-plane/subcommand/server-acl-init/rules_test.go

```diff
@@ -496,111 +498,251 @@ func TestSyncRules(t *testing.T) {
 			Expected: `node "k8s-sync" {
     policy = "write"
   }
-  node_prefix "" {
-    policy = "read"
+    node_prefix "" {
+      policy = "read"
+    }
+    service_prefix "" {
+      policy = "write"
+    }`,
+		},
+		{
+			Name:                           "Namespaces are disabled, non-default node name",
+			ConsulSyncDestinationNamespace: "sync-namespace",
```

Comment:
Even though there are defaults, should you be specific about all settings in the test. For example, Add 

```
EnablePartitions: "false",
PartitionName: "",
EnableNamespaces: "false",
```

to this test.

The only reason I bring this up is that I could see us or docs in the future grabbing one of these testcases and saying 'this matrix of settings, results in this'

Reply:
good call!! I think i fixed that if youd like to take a look.